serve_test.ts

```diff
@@ -48,6 +49,23 @@ const testCases: Array<testCase> = [
     method: Method.GET,
     expected: 'john',
   },
+  {
+    name: 'valid post',
+    registered: post('/params', ({ params }) => [200, params.name]),
+    path: 'params',
+    params: JSON.stringify({ name: 'ben' }),
+    method: Method.POST,
+    expected: 'ben'
+  },
+  {
+    name: 'valid post with detailed content-type',
+    registered: post('/params', ({ params }) => [200, params.name]),
+    path: 'params',
+    params: JSON.stringify({ name: 'tom' }),
+    headers: {['content-type']: 'application/json; charset=utf-8'},
```

Comment:
No brackets are needed here.
```suggestion
    headers: { 'content-type': 'application/json; charset=utf-8' },
```